docs/docs/user/migration-guide/v6-v7.md

```diff
@@ -9,54 +9,94 @@ As a result of the restructure, some imports need to be adjusted:
 **KPOps Python API**
 
 ```diff
--import kpops
-+import kpops.api as kpops
+- import kpops
++ import kpops.api as kpops
 ```
 
 **builtin KPOps components**
 
 ```diff
--from kpops.components import (
--    HelmApp,
--    KafkaApp,
--    KafkaConnector,
--    KafkaSinkConnector,
--    KafkaSourceConnector,
--    KubernetesApp,
--    StreamsBootstrap,
--    ProducerApp,
--    StreamsApp,
--    PipelineComponent,
--    StreamsApp,
--    ProducerApp,
--)
-+from kpops.components.base_components import (
-+    HelmApp,
-+    KafkaApp,
-+    KafkaConnector,
-+    KafkaSinkConnector,
-+    KafkaSourceConnector,
-+    KubernetesApp,
-+    PipelineComponent,
-+)
-+from kpops.components.streams_bootstrap import (
-+    StreamsBootstrap,
-+    StreamsApp,
-+    ProducerApp,
-+)
+- from kpops.components import (
+-     HelmApp,
+-     KafkaApp,
+-     KafkaConnector,
+-     KafkaSinkConnector,
+-     KafkaSourceConnector,
+-     KubernetesApp,
+-     StreamsBootstrap,
+-     ProducerApp,
+-     StreamsApp,
+-     PipelineComponent,
+-     StreamsApp,
+-     ProducerApp,
+- )
++ from kpops.components.base_components import (
++     HelmApp,
++     KafkaApp,
++     KafkaConnector,
++     KafkaSinkConnector,
++     KafkaSourceConnector,
++     KubernetesApp,
++     PipelineComponent,
++ )
++ from kpops.components.streams_bootstrap import (
++     StreamsBootstrap,
++     StreamsApp,
++     ProducerApp,
++ )
```

Comment:
Why did they get formated? 🤔

Reply:
I did that manually because I remembered that there should be one extra indentation space. I can move it to `v7` if it's confusing here, wdyt?